src/ucp/wireup/address.c

```diff
@@ -538,11 +545,13 @@ static void *ucp_address_unpack_md_info(const void *ptr,
                                         int *empty_dev)
 {
     uint8_t md_byte = *(uint8_t*)ptr;
+    size_t mask;
 
     *empty_dev = md_byte & UCP_ADDRESS_FLAG_MD_EMPTY_DEV;
+    mask       = (addr_version == UCP_OBJECT_VERSION_V1) ?
+                 UCP_ADDRESS_FLAG_MD_MASK_V1 : UCP_ADDRESS_FLAG_MD_MASK;
```

Comment:
maybe add function ucp_address_md_mask(version)
to reuse it in line 534 and 415